namespaces/cloud-platform-live-0.k8s.integration.dsd.io/offender-management-staging/resources/ecr-allocations-public.tf

```diff
@@ -0,0 +1,27 @@
+terraform {
+  backend "s3" {}
+}
```

Comment:
the `terraform {}` and `provider "aws" {}` blocks should only be [declared once](https://ministryofjustice.github.io/cloud-platform-user-docs/02-deploying-an-app/005-add-aws-resources/#usage).